_posts/2020-05-27-ols1-wrapup.md

```diff
@@ -0,0 +1,46 @@
+ 
+Some of the common themes that surprised us and brought us joy during the presentations: 
+- The COVID-19 pandemic had an unexpected impact of our program beyond the initial goals as our mentees could transfer their OLS skills in other areas like establishing effective communication among the international COVID collaborations, designing the format of their lab meetings online, setting mentor-mentee relationships in their local networks, and connecting folk musicians and artists with each other in Nepal during the lockdown.
+- The [open canvas](https://docs.google.com/presentation/d/1MeJo0TyuMg_waLk1J4q9y1aAqKNMuRBlnmxEChSz-cQ/edit#slide=id.p) exercise was commonly useful for everyone. We presented this in [our first cohort call](https://openlifesci.org/ols-1/week02/), but a large number of mentees found it so foundational they shared it even 3-4 months later in their presentations. 
+- Skills like collaborative work culture, self care, ally skills, giving and receiving feedback, agile management, and knowledge dissemination are valuable beyond research.  
+A Youtube Stream can make all the difference in helping your friends and family attend important events: 
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">The graduation was even more special because my <a href="https://twitter.com/hashtag/VUamsterdam?src=hash&amp;ref_src=twsrc%5Etfw">#VUamsterdam</a> family, <a href="https://twitter.com/gravana?ref_src=twsrc%5Etfw">@gravana</a>, and <a href="https://twitter.com/PhDToothFAIRy?ref_src=twsrc%5Etfw">@PhDToothFAIRy</a><br>were there 😍<br>AND, my mother watched my graduation in real-time. She had to miss both my wedding and PhD defense. It&#39;s the first event in my life abroad that she witnessed 👩‍👧 <a href="https://t.co/xWWaJ9kLhw">pic.twitter.com/xWWaJ9kLhw</a></p>&mdash; Lena Karvovskaya (@LangData) <a href="https://twitter.com/LangData/status/1263468708330078209?ref_src=twsrc%5Etfw">May 21, 2020</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script> 
+
+## Announcing OLS-2 - please share with your networks! 
+
+Motivated by the overwhelming success of OLS so far, we’re incredibly excited to announce that applications for round 2 of OLS are now officially open!  
+Whether you were excited by OLS-1 but missed the deadline, or whether you’ve heard about it for the first time today, you’re welcome to apply. 
+
+To apply, please visit [EasyChair](https://easychair.org/my/conference?conf=ols2). We also have [templates](https://github.com/open-life-science/application-forms) you can download to use when preparing your application. 
```

Comment:
We could use here the timeline as on the index page:

![Screenshot 2020-05-28 at 10 04 12](https://user-images.githubusercontent.com/1842467/83115426-a167e500-a0ca-11ea-8e24-a3db0577fd32.png)

What do you think?

It can be done with `{% include _includes/ols-2-timeline.md %}`